account-kit/signer/src/client/index.ts

```diff
@@ -425,7 +426,7 @@ export class AlchemySignerWebClient extends BaseSignerClient<ExportWalletParams>
    */
   public override oauthWithPopup = async (
     args: Extract<AuthParams, { type: "oauth"; mode: "popup" }>,
-  ): Promise<User | AuthLinkingPrompt> => {
+  ): Promise<User | AuthLinkingPrompt | IdTokenOnly> => {
```

Comment:
do you feel this is getting out of hand / can be improved in v5?

Reply:
Note at least that this thing is "internal" because we don't expect outsiders to be dealing with the web client.